components/org.apache.stratos.manager.console/console/themes/theme0/partials/applications_deploy.hbs

```diff
@@ -22,7 +22,7 @@
 <div id="centered">
     <div class="row title">
         <div class="title-main text-center">
-            <h1>Create Deployment Policy for Application {{applicationAlias}}</h1>
+            <h1>Deploy Deployment Policy for Application {{applicationAlias}}</h1>
```

Comment:
There is another issue in this as well. When the button is clicked, it's text changes to Add Application Policy and Add Deployments. Try to change that as well.